# Section: KERNEL PARAMETERS
# Reference: broker1 (all other nodes diffed against this)

## broker1 vs connect
--- broker1
+++ connect
@@ -16,7 +16,7 @@
 net.core.netdev_max_backlog = 5000
 net.ipv4.tcp_rmem = 4096	1048576	16777216
 net.ipv4.tcp_wmem = 4096	1048576	16777216
-net.ipv4.tcp_max_syn_backlog = 4096
+net.ipv4.tcp_max_syn_backlog = 2048
 
 --- Transparent Huge Pages ---
 enabled: always [madvise] never

## broker1 vs monitor
--- broker1
+++ monitor
@@ -16,7 +16,7 @@
 net.core.netdev_max_backlog = 5000
 net.ipv4.tcp_rmem = 4096	1048576	16777216
 net.ipv4.tcp_wmem = 4096	1048576	16777216
-net.ipv4.tcp_max_syn_backlog = 4096
+net.ipv4.tcp_max_syn_backlog = 2048
 
 --- Transparent Huge Pages ---
 enabled: always [madvise] never

